lib/src/fade.dart

```diff
@@ -173,9 +159,8 @@ class _RotatingTextState extends State<FadeAnimatedTextKit>
                 animation: _controller,
                 builder: (BuildContext context, Widget child) {
                   return Opacity(
-                    opacity: !(_fadeIn.value == 1.0)
-                        ? _fadeIn.value
-                        : _fadeOut.value,
+                    opacity:
+                        _fadeIn.value != 1.0 ? _fadeIn.value : _fadeOut.value,
                     child: Text(

```

Comment:
@awhitford Make this `Text` widget a child of the `AnimatedBuilder` and reference it using the child property given in the builder function.

Reply:
Good idea.